a10_octavia/common/utils.py

```diff
@@ -53,6 +53,7 @@ def validate_partial_ipv4(address):
     if not netaddr.valid_ipv4('.'.join(octets), netaddr.core.INET_PTON):
         raise cfg.ConfigFileValueError('Invalid partial IPAddress value given'
                                        ' in configuration: {0}'.format(address))
+    return address
```

Comment:
`tests/unit/common/test_utils.py` need to be tested also for this return change added.